jams/schemata/namespaces/pitch/note_midi.json

```diff
@@ -0,0 +1,8 @@
+{"note_midi":
+    {
+        "value": { "type": "number" },
```

Comment:
I suppose if we want the spec to conform to MIDI standards, it should be a 7-bit integer [0-127].  But if we want to support fractional numbers, [0, 128), that breaks things a little.

Maybe @craffel can chime in here?